frontend/components/sections/lead-form.js

```diff
@@ -0,0 +1,95 @@
+import Button from '../elements/button';
+import { useState } from 'react';
+import { fetchAPI } from 'utils/api';
+import * as yup from 'yup';
+import { Formik } from 'formik';
+
+const LeadForm = ({ data }) => {
+  const [loading, setLoading] = useState(false);
+
+  const schema = yup.object().shape({
+    email: yup.string().email(),
+  });
+
+  return (
+    <div className="py-10 text-center">
+      <h1 className="text-3xl mb-10 font-bold mb-2">{data.title}</h1>
+      <div className="flex flex-col items-center flex-wrap ">
+        <Formik
+          initialValues={{ email: '' }}
+          validate={async (values) => {
+            const valid = await schema.isValid({
+              email: values.email,
+            });
+
+            const errors = {};
+
+            if (!values.email) {
+              errors.email = 'Email is required'
+            }
+
+            if (!valid) {
+              errors.email = 'Please enter a valid email';
+            }
+
+            return errors;
+          }}
+          onSubmit={async (values, { setSubmitting, setErrors }) => {
+            setLoading(true);
+            try {
+              await fetchAPI('/lead-form-submissions', {
```

Comment:
You might need to remove the api errors here.

Reply:
👍  you mean something like ```setErrors({ api: null })``` ?